src/types.rs

```diff
@@ -524,6 +524,7 @@ pub enum ObjectType {
     Elf,
     Macho,
     Pe,
+    Symbolic,
```

Comment:
This is deprecated and no longer supported.

Reply:
"Symbolic" effectively means unknown.

Reply:
Want me to change the test assertions? This would otherwise no longer roundtrip the original value (symbolic).

Reply:
Yes. It's probably even better if symbolicator explicitly indicates that it doesn't know the object type. Sentry should have special handling for `"unknown"`, let me double check. If not, we can add it.

Reply:
Ah, I misremembered and we only handle unknown architectures. Adding `type` there, too.

Reply:
https://github.com/getsentry/sentry/pull/15740